backend/lib/endpoints/schema/organizations.js

```diff
@@ -3,22 +3,26 @@ const { strictSchema } = require("./utils");
 const { locationSchema } = require("./location");
 
 const organization = {
-  about: S.string(),
+  about: S.string().maxLength(100),
   email: S.string().format("email"),
   global: S.boolean(),
   industry: S.string(),
   language: S.string(),
   location: locationSchema,
   name: S.string(),
-  needs: S.array().maxItems(10).items(S.string()),
+  needs: S.object()
+    .prop("volunteers", S.boolean().required().default(false))
+    .prop("donations", S.boolean().required().default(false))
+    .prop("staff", S.boolean().required().default(false))
+    .prop("other", S.boolean().required().default(false)),
   ownerId: S.string(),
   type: S.string(),
-  url: S.object()
-    .prop("appStore", S.string().format("url"))
-    .prop("linkedin", S.string().format("url"))
-    .prop("playStore", S.string().format("url"))
-    .prop("twitter", S.string().format("url"))
-    .prop("website", S.string().format("url")),
+  urls: S.object()
```

Comment:
we could probably reuse the `urls` from the individual users, at least some of the validation rules
but it also seemed like the linkedIn URL wasn't correct